pkg/etcdutil/layout.go

```diff
@@ -64,12 +63,12 @@ func TaskMasterPath(appName string, taskID uint64) string {
 	return path.Join("/", appName, TasksDir, strconv.FormatUint(taskID, 10), TaskMaster)
 }
 
-func MetaPath(linkType, appName string, taskID uint64) string {
+func MetaPath(appName string, taskID uint64) string {
 	return path.Join("/",
 		appName,
 		TasksDir,
 		strconv.FormatUint(taskID, 10),
-		linkType)
+		"meta")
```

Comment:
My idea is that I don't want to do the work here to also define link type.

My purpose to make sure the Master can send epoch notification to all workers.